### PR TITLE
Feature(fileName): Add additional option to include a file name separate from the title.

### PR DESCRIPTION
**Pull request by jbavari**
First off, thank you very much for developing this tool. It's worked great.

I've thrown together a quick PR to add the option of passing a fileName to name the Mac OSX DMG.

If no 'fileName' is provided, use the title as before.

Ran tests and they still pass.

### CODE REVIEW

**Comment by stefanjudis** [comment]
Hey hey, thanks.

Currently I'm on vacation, but will have a look next week. :bowtie:

**Comment by stefanjudis** [comment]
@jbavari 

Hey just had a quick look.
If we want to add this functionality we should expose it to all platforms, shouldn't we?

**Comment by jbavari** [comment]
@stefanjudis Absolutely. You want me to add that in as well? Happy to do so.

**Comment by stefanjudis** [comment]
@jbavari 

Cool. Yeah absolutely. 

One thing that's coming to my mind immediatelly is to unify the configs somehow. 
But this might be a follow up or even a completely different ticket.

But anyways, any contribution is more than welcome. :bowtie: So please go ahead if you've got time.

**Comment by jbavari** [comment]
@stefanjudis I went ahead and updated the changes for Windows setup executable as well. Cheers!

**Comment by stefanjudis** [comment]
@jbavari 

Looks good to me so far...
Will make a final check over the weekend. 

Thanks a lot. :bowtie:

File: lib/osx.js [hunk 1]
```diff
@@ -50,9 +50,10 @@ var OSXBuilder = {
     options.log( 'Kicking off ´appdmg´' );
 
     var appdmg = require( 'appdmg' );
+    var fileName = options.config.osx.fileName ? options.config.osx.fileName : options.config.osx.title;
```
Comment (stefanjudis) [inline lib/osx.js]: Can we add a line break after `?` and `:` here.
I'm a fan of the 80 characters per line if possible. :blush:

File: lib/win.js [hunk 2]
```diff
@@ -27,11 +27,16 @@ var WinBuilder = {
   build : function( options, callback ) {
     options.log( '- Starting build for ´' + options.platform + '´ - ' );
 
-    var configFilePath = writeConfigFile( 'installer.nsi', {
-      appPath : _windowsify( options.appPath ),
-      name    : options.config.win.title,
-      out     : _windowsify( options.out )
-    } );
+    var fileName = options.config.win.fileName ? options.config.win.fileName : [ options.config.win.title, 'Setup' ].join( ' ' );
```
Comment (stefanjudis) [inline lib/win.js]: Same here.

**Comment by stefanjudis** [comment]
I only have a few coding style comments. Looks good to me.

The update of the readme file is missing though. :)

After that I think we're ready to go. :)

Thanks for the effort.

**Comment by stefanjudis** [comment]
As @AlicanC points out in #28 it might make sense to also to make the shortcuts configurable.

What do you folks think?

@jbavari

**Comment by jbavari** [comment]
@stefanjudis Yes, I'd agree, those configuration's would be awesome.

**Comment by stefanjudis** [comment]
@jbavari 

Okay - cool. Do you mind adding these, too? Because this PR is in better shape then the other one as it includes both platforms. :)

Sorry for this long pending PR - but I'd like to take it. :bowtie:

**Comment by stefanjudis** [comment]
@jbavari Any news here?

**Comment by stefanjudis** [comment]
@jbavari 

I'll copy your changes in my own branch then in a few days and will close this here.

**Comment by havenchyk** [comment]
Can this issue be closed? Otherwise @jbavari do you have any updates?

**Comment by stefanjudis** [comment]
Closed due to inactivity.

**Pull request closed without merging**